profiling/src/exporter/mod.rs

```diff
@@ -162,12 +162,10 @@ impl ProfileExporter {
     /// "RFC: Pprof System Info Support".
     pub fn build(
```

Comment:
What are the ownership semantics after a `profile` gets passed in here? Do I still need to drop it separately? Should I assume the request now owns the object? What happens if the export fails?

Reply:
We take it by value, so its consumed.

Reply:
noted in comments

Reply:
Actually, it occurred to me -- what about failures? Are we forced to throw the profile away of the reporting fails? For Ruby we never retry, but I'm not sure if any other profilers have logic for retrying.

Reply:
Thanks for checking!